internal/security/proxy/service.go

```diff
@@ -156,16 +178,6 @@ func (s *Service) Init() error {
 		}
 	}
 
-	err := s.initAuthMethod(s.configuration.KongAuth.Name, s.configuration.KongAuth.TokenTTL)
-	if err != nil {
-		return err
-	}
-
-	err = s.initACL(s.configuration.KongACL.Name, s.configuration.KongACL.WhiteList)
-	if err != nil {
-		return err
-	}
```

Comment:
Why are these no longer needed?

Reply:
part of @beaufrusetta 's PR: my guess is since we are going to use JWT file, this duplication is no longer needed; 
@bnevis-i 
 and @beaufrusetta do yo agree?